Geometry/TrackerCommonData/data/PhaseI/pixfwdCylinder.xml

```diff
@@ -56,11 +56,20 @@ root volume plus an anchor point for positioning.
  <Constant name="CylindersPortCardsLength1" value="533.7*mm"/><!--523.0 mm-->
  <Constant name="CylindersPortCardsThickness" value="18.9*mm"/><!--5.40 mm-->
 
- <Constant name="CylindersServiceZoff" value="320.0*mm"/>
+ <Constant name="CylindersServiceZoff" value="[pixfwd:AnchorZ]+[pixfwd:Disk3Z]+[pixfwdDisks:DiskHalfWidth]"/>
  <Constant name="CylindersServiceRmin" value="100*mm"/>
  <Constant name="CylindersServiceZ0"   value="0*mm"/>
+ <Constant name="CylindersServiceZext"   value="-4*mm"/>
  <Constant name="CylindersServiceZMin" value="[CylindersServiceZoff]"/>
  <Constant name="CylindersServiceZMax" value="[CylindersOuterLength]"/>
+
+ <Constant name="CarbonFiberSheetNearIPRmin" value="101.0*mm"/>
```

Comment:
@Michael-Krohn , @krose13 : is this 101mm a *real* value or one caused by limited size of the service cylinder? If this is only due to constraints, we can make it larger now.